build.sbt

```diff
@@ -40,7 +40,7 @@ addCommandAlias(
   ";mockNative/compile"
 )
 
-val zioVersion = "2.0.0-RC3"
+val zioVersion = "2.0.0-RC4+33-399191a6-SNAPSHOT"
```

Comment:
So we can't release an RC4 version, have to wait for RC5?

Reply:
If possible to release a more limited version for RC4 as we discussed, it would be nice to split the PR. If we only release zio-mock for RC5 then downstream dependencies and end users must wait for RC5 support in everything before being able to use it (for example I can only update zio-aws to RC5 once all its deps are on RC5, and have to skip RC4 completely)

Reply:
Forget it, RC5 is out :)